packages/core/bootstrap/src/lib/ws/types.ts

```diff
@@ -0,0 +1,30 @@
+
+export interface WSConfig {
+  /** Info about the connection */
+  connectionInfo: {
+    key: string
+  }
+  /** Maximum amount of parallel connections */
+  connectionLimit: number
```

Comment:
I don't see this used? We should populate the limit from our rate limits package, or it's set manually by the user.

Reply:
#467